client/platform/desktop/frontend/components/Recent.vue

```diff
@@ -111,29 +123,90 @@ export default defineComponent({
       return 'mdi-image-multiple';
     }
 
+    async function preloadCheck(recent: JsonMetaCache) {
+      //Attempts to preload the data to see if there are any isues
+      try {
+        await api.checkDataset(recent.id);
+      } catch (e) {
+        const result = await prompt({
+          title: 'Error Loading Data',
+          text: e,
+          confirm: true,
+          positiveButton: 'Delete',
+          negativeButton: 'Cancel',
```

Comment:
Maybe don't allow someone to delete a dataset from this menu.  Force them thorough the regular delete menu and make them look at the scary warnings